.github/workflows/rolling-release.yml

```diff
@@ -3,26 +3,30 @@ name: Create rolling release
 on:
   pull_request:
     paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
       - '**.md'
-      - 'appveyor.yml'
+      - '*.yml'
       - 'scripts/*'
-      - '.github/ISSUE_TEMPLATE/*'
   push:
     branches:
       - master
       - dev
     paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
       - '**.md'
-      - 'appveyor.yml'
+      - '*.yml'
       - 'scripts/*'
-      - '.github/ISSUE_TEMPLATE/*'
   workflow_dispatch:
 
 jobs:
-  windows-build:
+  build-windows:
     runs-on: windows-2019
+    env:
+      POWERSHELL_TELEMETRY_OPTOUT: 1
     steps:
-    - uses: actions/checkout@v2.3.1
+    - uses: actions/checkout@v2
```

Comment:
Is there a reason for moving to the latest v2 tag rather than staying on specifically tagged versions and upgrading as needed?

Reply:
1. Won't have to be manually upgraded.
2. Breaking changes are limited to major version revisions.

Reply:
This is probably okay.